libs/ledgerjs/packages/cryptoassets/data/eip712.js

```diff
@@ -0,0 +1 @@
+module.exports = {}
```

Comment:
Nothing here yet?

Reply:
Yup expected since we don't have the CAL ready yet 👌 
That would also be the result if there were no messages in the CAL